sample/src/main/java/com/metova/privvy/sample/ui/floatingnumber/number/NumberRouter.java

```diff
@@ -19,5 +19,6 @@
     public void navigateToList() {
         Intent intent = new Intent(getActivity(), ListActivity.class);
         getActivity().startActivity(intent);
+        getActivity().finish();
```

Comment:
I know you took this out, but not having it meant that events were happening N amount of times, where N = the number of different instances of the same activity were up. So a button click would cause the presenter to update the number 4 times, for instance.